cloudify_agent/installer/config/configuration.py

```diff
@@ -307,8 +307,7 @@ def installation_attributes(cloudify_agent, runner):
                 # 'pwd' module does not exists in a windows python
                 # installation.
                 # TODO - maybe use some environment variables heuristics?
```

Comment:
if we're not installing into the home dir anymore, this comment is obsolete, i suppose remove it